src/main/java/net/glowstone/inventory/crafting/CraftingManager.java

```diff
@@ -459,23 +459,24 @@ private void loadRecipes() {
         // Load recipes from recipes.yml file
         InputStream in = getClass().getClassLoader().getResourceAsStream("builtin/recipes.yml");
         if (in == null) {
-            GlowServer.logger.warning("Could not find default recipes on classpath");
+            LocalizedStrings.Console.Warn.Recipe.NO_DEFAULTS.log();
             return;
         }
 
         ConfigurationSection config = YamlConfiguration
                 .loadConfiguration(new InputStreamReader(in));
 
         // shaped
-        for (Map<?, ?> data : config.getMapList("shaped")) {
-            ItemStack resultStack = ItemStack.deserialize((Map<String, Object>) data.get("result"));
+        for (Map<?, ?> data : config.getMapList("shaped")) { //NON-NLS
```

Comment:
You used `// NON-NLS` (with a space) in the rest of the project, could you be consistent with this PR?